### PR TITLE
Migrate hub to gh

### PR DESCRIPTION
**Pull request by haslinghuis**
See: https://github.com/actions/runner-images/issues/8362

Usage: https://cli.github.com/manual/gh_release_edit

### CODE REVIEW

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
